forms/form-british-citizenship.php

```diff
@@ -26,14 +26,23 @@ function return_form_british_citizenship( $content ) {
 	                        <p>Enter the details of the certificate holder at the time the certificate was issued.</p>
 	                    </div>
 	                    <div class="form-row">
-	                        <label for="certificate_name">Full name</label>
-	                        <input type="text" id="certificate_name" name="certificate-name" aria-required="true" required ' . set_value( 'certificate-name' ) . '>
-	                        ' . field_error_message( 'certificate-name', 'Certificate holder name' ) . '
+	                        <label for="certificate_first_name">Certificate holder first name</label>
+	                        <input type="text" id="certificate_first_name" name="certificate-first-name" ' . set_value( 'certificate-first-name' ) . '>
```

Comment:
Should this field have the `required` and `aria-required="true"` attributes that were on the `certificate_name` field?